src/main/java/edu/ucsb/cs156/example/repositories/RecommendationRequestRepository.java

```diff
@@ -0,0 +1,10 @@
+package edu.ucsb.cs156.example.repositories;
+
+import edu.ucsb.cs156.example.entities.RecommendationRequest;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+public interface RecommendationRequestRepository extends CrudRepository<RecommendationRequest, Long>{
+    Iterable<RecommendationRequest> findAllById(int id);
```

Comment:
You shouldn't need to define findAllById here since it is part of the CrudRepository interface. You can just leave the body of this interface blank